provider/filestore_test.go

```diff
@@ -168,147 +185,60 @@ func testPathPrefix(t *testing.T, store FileStore, storeName string) {
 	}
 }
 
-func getFileStores() (map[string]FileStore, error) {
-	localFileStore, err := getLocalFileStore()
-	if err != nil {
-		return nil, fmt.Errorf("could not get local filestore: %v", err)
-	}
-
-	s3FileStore, err := getS3FileStore()
-	if err != nil {
-		return nil, fmt.Errorf("could not get s3 filestore: %v", err)
-	}
-
-	azureFileStore, err := getAzureFileStore()
-	if err != nil {
-		return nil, fmt.Errorf("could not get azure filestore: %v", err)
-	}
-
-	gcsFileStore, err := getGCSFileStore()
-	if err != nil {
-		return nil, fmt.Errorf("could not get gcs filestore: %v", err)
-	}
-
-	fileStoresMap := map[string]FileStore{
-		"local": localFileStore,
-		"s3":    s3FileStore,
-		"azure": azureFileStore,
-		"gcs":   gcsFileStore,
-	}
-
-	return fileStoresMap, nil
-}
-
-func getLocalFileStore() (FileStore, error) {
-	config := pc.LocalFileStoreConfig{
-		DirPath: "file:///tmp/",
-	}
-
-	serializedConfig, err := config.Serialize()
-	if err != nil {
-		return nil, fmt.Errorf("could not serialized local config: %s", err)
-	}
-	fileStore, err := NewLocalFileStore(serializedConfig)
-	if err != nil {
-		return nil, fmt.Errorf("could not initialize local filestore: %s", err)
-	}
-
-	return fileStore, nil
-}
-
-func getS3FileStore() (FileStore, error) {
-	s3Config := pc.S3FileStoreConfig{
-		Credentials: pc.AWSCredentials{
-			AWSAccessKeyId: os.Getenv("AWS_ACCESS_KEY_ID"),
-			AWSSecretKey:   os.Getenv("AWS_SECRET_KEY"),
-		},
-		BucketRegion: os.Getenv("S3_BUCKET_REGION"),
-		BucketPath:   os.Getenv("S3_BUCKET_PATH"),
-		Path:         "featureform-unit-test",
-	}
-
-	serializedConfig, err := s3Config.Serialize()
-	if err != nil {
-		return nil, fmt.Errorf("could not serialized s3 config: %s", err)
-	}
-	fileStore, err := NewS3FileStore(serializedConfig)
-	if err != nil {
-		return nil, fmt.Errorf("could not initialize s3 filestore: %s", err)
-	}
-
-	return fileStore, nil
-}
-
-func getHDFSFileStore() (FileStore, error) {
-	hdfsConfig := pc.HDFSFileStoreConfig{
-		Host:     "localhost",
-		Port:     "9000",
-		Username: "hduser",
-		Path:     "/tmp/",
-	}
-
-	serializedConfig, err := hdfsConfig.Serialize()
-	if err != nil {
-		return nil, fmt.Errorf("could not serialized hdfs config: %s", err)
-	}
-	fileStore, err := NewHDFSFileStore(serializedConfig)
+func TestDoesSourceUrlExist(t *testing.T) {
+	err := godotenv.Load("../.env")
 	if err != nil {
-		return nil, fmt.Errorf("could not initialize hdfs filestore: %s", err)
+		t.Fatalf("could not load env: %s", err)
```

Comment:
I wouldn't do this because the CI/CD doesn't have a `.env` file so it will fail.

Reply:
so will this work in ci/cd?